test/lib/benchmark.js

```diff
@@ -11,6 +11,13 @@ exports.createBenchmark = (opts) => {
   return new Benchmark(opts)
 }
 
+/**
+ * Represents the benchmark test
+ * @class
+ * @param {object} opts test suite configuration options
+ * @param {string} opts.name name of test suite
+ * @param {number} opts.runs number of iterations to run. 1000 by default
```

Comment:
You can use the dsl to indicate the default:

```suggestion
 * @param {number} [opts.runs=1000] number of iterations to run.
```

Looks like the `name` is optional as well.

https://jsdoc.app/tags-param#optional-parameters-and-default-values